{{cookiecutter.app_name}}/.github/workflows/test.yml

```
@@ -14,10 +14,16 @@ jobs:
         with:
           go-version: 1.16.x
 
+      - name: Create ENV file
+        run: cp ".env.test" ".env"
```

Why is there a need to do this copy? The workflow can set the environment variable to test:

```suggestion
        run: export ENV=test
```

Then in the Makefile `include .env.test` would work 💭

---

Because `Makefile` requires `.env` file by default
```make
include .env
ifdef ENV
include .env.$(ENV)
endif
```

I decide not to use `-include .env` because I want command to raise an error if `.env` file is missing. It is better to know the error as fast as possible.

---

But then, in that case, shouldn't the `.env.example` be used? 

```
run: cp ".env.example" ".env"
```

I find it confusing to rename `.env.test` to `.env`

---

Good catch. Fixed in cebb025525dbb4eec2dcc231a93d36a73f05895b